controller/payload/manager.py

```diff
@@ -39,3 +42,43 @@ def update_payload_status(
     project_id: str, payload_id: str, status: str
 ) -> InformationSourcePayload:
     return payload.update_status(project_id, payload_id, status)
+
+
+def get_labeling_function_on_10_records(
+    project_id: str, information_source_id: str
+) -> LabelingFunctionSampleRecords:
+    doc_bin_samples, sample_records = payload_scheduler.prepare_sample_records_doc_bin(
+        project_id=project_id, information_source_id=information_source_id
+    )
+    (
+        calculated_labels,
+        container_logs,
+        code_has_errors,
+    ) = payload_scheduler.run_labeling_function_exec_env(
+        project_id=project_id,
+        information_source_id=information_source_id,
+        prefixed_doc_bin=doc_bin_samples,
+    )
+    calculated_labels = fill_missing_record_ids(sample_records, calculated_labels)
+
+    return LabelingFunctionSampleRecords(
+        records=[
+            LabelingFunctionSampleRecordWrapper(
+                record_id=record_item[0],
+                full_record_data=record_item[1],
+                calculated_labels=calculated_labels[record_item[0]],
+            )
+            for record_item in sample_records
+        ],
+        container_logs=container_logs,
+        code_has_errors=code_has_errors,
+    )
+
+
+def fill_missing_record_ids(sample_records: List[str], calculated_labels) -> List[str]:
+    for record_item in sample_records:
+        record_id = record_item[0]
+        if record_id not in calculated_labels.keys():
```

Comment:
since caluclated labels is a dict a lookup without .keys would be better
https://stackoverflow.com/a/4731079/19801189